docs/sign-up.rst

```diff
@@ -0,0 +1,72 @@
+.. _sign-up:
+
+
+===================================
+Create a CrateDB Cloud user account
+===================================
+
+This page shows you how to:
+
+* Create a user account
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _create-account:
+
+Create an account
+=================
+
+Call up the `CrateDB Cloud sign-up page`_ and complete the form.
+
+.. image:: _assets/img/cloud-sign-up-page.png
+
+Once you've completed the form, select *Sign up*.
+
+You'll receive a confirmation email providing your verification code.
+
+Open your email inbox and open the verification email.
+
+.. NOTE::
+
+  If you didn't receive the verification email, select *Resend it* to request a
+  new verification email.
+
+Copy your verification code from the email.
+
+.. image:: _assets/img/confirmation-email-received.png
+
+Paste your verification code into the corresponding field on the sign-up page
+and select *Confirm Account* to complete the CrateDB Cloud user account
+registration.
+
+.. image:: _assets/img/enter-verification-code.png
+
+Once you've signed up, you're automatically logged into CrateDB Cloud.
+
+Your browser should redirect you to the `CrateDB Cloud Console`_.
+
+.. NOTE::
+
+   CrateDB Cloud ships with the Cloud Console, a dedicated web administration
+   interface for interacting with CrateDB Cloud.
+
+   To learn what the different Cloud Console views look like and what its
+   features can do, check out the `Cloud Console Reference`_.
+
+.. image:: _assets/img/cloud-console-after-sign-up.png
+
+
+Next steps
+==========
+
+Now that you've signed up to CrateDB Cloud, it's time to install Croud, the
+`dedicated command-line interface (CLI)`_.
+
+.. _crateDB cloud sign-up page: http://bregenz.a1.cratedb.cloudhttps://crate.auth.eu-west-1.amazoncognito.com/signup?redirect_uri=https%3A%2F%2Fbregenz.a1.cratedb.cloud%2Foauth2%2Flogin%2Fcallback&state=d0f5a031-7421-4062-9133-55cb75d17d42&response_type=code&client_id=7gnpivbvackgcmqat9r0m8p4b1
```

Comment:
Please don't use this link. It is different for every new user. 
Please just link to either eastus2.azure.cratedb.cloud or westeurope.azure.cratedb.cloud. The redirect will happen automatically.